fossunited/foss_hackathon/doctype/foss_hackathon_team/foss_hackathon_team.py

```
@@ -41,7 +41,20 @@ def validate(self):
         self.validate_new_members_not_in_other_teams()
 
     def on_update(self):
-        self.delete_if_empty_team()
+        self.update_member_team_links()
+        self.auto_delete_if_empty()
+
+    def on_trash(self):
+        """Clear team link when team is deleted"""
+        for participant in self.members:
+            if member.member:
```

Ruff (0.15.5)

[error] Undefined name `member`

(F821)